queries/tv.py

```diff
@@ -0,0 +1,166 @@
+"""
+
+    Reynir: Natural language processing for Icelandic
+
+    Television schedule query response module
+
+    Copyright (C) 2019 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see http://www.gnu.org/licenses/.
+
+
+    This module handles queries related to television schedules.
+    Only handles RÚV (for now).
+
+"""
+
+# TODO: Support TV schedule queries for other stations than RÚV
+
+import logging
+import re
+from datetime import datetime, timedelta
+from queries import query_json_api, gen_answer
+
+
+_TELEVISION_QTYPE = "Television"
+
+
+# This module wants to handle parse trees for queries
+HANDLE_TREE = True
+
+# The context-free grammar for the queries recognized by this plug-in module
+GRAMMAR = """
+
+Query →
+    QTelevision
+
+QTelevision → QTelevisionQuery '?'?
+
+QTelevisionQuery →
+    QTVWhatIsNom QTVEiginlega? QTVGoingOn? QTVOnTV QTVNow?
+    | "hvað" "er" QTVEiginlega? QTVBeingShown? QTVOnTV QTVNow?
```

Comment:
"hvað" "er" could equivalently be added to `QTVWhatIsDative` and this line deleted